vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Handling Usage Files
+description: Guidance for handling usage files.
+ms.date: 05/11/2023
+---
+
+# Handling Usage Files
```

Comment:
```suggestion
# Supplying Usage for Packages
```

Reply:
This document should cover usage info as a topic, not just specifically usage files.

For example, it should describe our existing heuristics for CMake config files & header only libraries.